starter-code/index.html

```diff
@@ -1,42 +1,95 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
 </head>
-<body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
 
-HD Music
-Listen to music as if you
-were listening live
+<body>
+  <div class="container">
+    <nav class="nav-bar">
+      <a href="#"><img class="logo" src="./images/spotify-logo.png" alt="" srcset=""></a>
+      <ul class="nav">
+        <li><a href="#">Premium</a></li>
+        <li><a href="#">Discover</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Download</a></li>
+      </ul>
+    </nav>
+    <div class="banner">
+      <div class="title-text">
+        <h1>Music for everyone.</h1>
+        <h2 class="subtitle">Spotify is now free on mobile, tablet and computer.
+          Listen to the right music, wherever you are.</h2>
+      </div>
+    </div>
+    <div class="features">
+      <h3 class="section-title">
+        What’s on Spotify?
+      </h3>
+      <ul class="feature-list">
+        <li class="feature"><img src="images/music-icon.png" alt="Music">
+          <h4 class="feature-title">Millions of Songs</h4>
+          <p class="feature-description">There are millions of
+            songs on Spotify</p>
+        </li>
+        <li class="feature"><img src="images/high-quality-icon.png" alt="High quality">
+          <h4 class="feature-title">HD Music</h4>
+          <p class="feature-description">Listen to music as if you
+            were listening live</p>
+        </li>
+        <li class="feature"><img src="images/devices-icon.png" alt="Devices">
+          <h4 class="feature-title">Stream Everywhere</h4>
+          <p class="feature-description">Stream music on your
+            smartphone, tablet or
+            computer</p>
+        </li>
+      </ul>
```

Comment:
With flexbox you can get rid of some tags..ul and li are not necessarily needed. one div (flex) with a-tags insode can do the job:)